### ISSUE TITLE
Add link to desktop releases

### ISSUE THREAD

**Issue by Miepee** [ajbura/cinny-site#5]
I was initially under the impression that cinny was only available as a web client.
The site should add a link to the [downloads of the desktop client](https://github.com/cinnyapp/cinny-desktop/releases), in order to show that desktop versions are indeed available.

**Comment by br4yd** [comment]
I just created a PR for adding that button, because I thought the same. 

https://github.com/cinnyapp/cinny-site/pull/9

**Comment by chrisdedman** [comment]
Bump this. I think it would make it easier for people to get to the desktop client when coming from Cinny website directly rather than matrix.

**Comment by Thales3006** [comment]
Also bumping this. My friend group almost gave up on the idea of using Cinny/Matrix because of the lack of a clear download button/link.

**Comment by chrisdedman** [comment]
> Also bumping this. My friend group almost gave up on the idea of using Cinny/Matrix because of the lack of a clear download button/link.

Don't quote me on that, but I believe Cinny abandoned the desktop version. I stopped using Cinny a while ago, but it was a conversation they were having.

**Comment by kfiven** [comment]
No, we didn't not abandon the desktop app. We didn't link because desktop app is not at par with web one and once it reaches there, it'll be properly linked.
